numpy/lib/function_base.py

```diff
@@ -2965,7 +2965,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
 
     >>> x = [-2.1, -1,  4.3]
     >>> y = [3,  1.1,  0.12]
-    >>> X = np.vstack((x,y))
+    >>> X = np.concatenate((x,y), axis=0)
```

Comment:
> I don't understand the issue here.

The issue is that if I run your code, I get a different result to what was here before:
```python
>>> x = [-2.1, -1,  4.3]
>>> y = [3,  1.1,  0.12]
>>> np.vstack((x,y))   # before
array([[-2.1 , -1.  ,  4.3 ],
       [ 3.  ,  1.1 ,  0.12]])
>>> np.concatenate((x,y), axis=0)  # yours (wrong)
array([-2.1 , -1.  ,  4.3 ,  3.  ,  1.1 ,  0.12])
>>> np.stack((x,y), axis=0)
array([[-2.1 , -1.  ,  4.3 ],  # charris (correct)
       [ 3.  ,  1.1 ,  0.12]])
```